tef-impl/src/main/java/flipkart/tef/execution/FlowBuilder.java

```diff
@@ -105,7 +132,7 @@ BiMap<DataAdapterKey<?>, Class<? extends IDataBizlogic<?>>> getDataAdapterMap()
     }
 
     @VisibleForTesting
-    Multimap<Class<? extends IBizlogic>, DataAdapterKey<?>> getDataDependencyMap() {
+    Multimap<Class<? extends IBizlogic>, DataDependencyDetail> getDataDependencyMap() {
```

Comment:
Some change in contract?

Reply:
Yes, though this method is visible only to tests.